<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 0)</string>
  </property>
  <widget class="QPushButton" name="HighScore">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>900</y>
     <width>1000</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: italic 18pt &quot;Harlow Solid Italic&quot;;</string>
   </property>
   <property name="text">
    <string>High Score</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitGame">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>970</y>
     <width>1000</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: italic 18pt &quot;Harlow Solid Italic&quot;;</string>
   </property>
   <property name="text">
    <string>Exit Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartGame">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>690</y>
     <width>1000</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: italic 18pt &quot;Harlow Solid Italic&quot;;</string>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1921</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 120pt &quot;Harlow Solid Italic&quot;;
color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string>Space Invaders</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Alien">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>310</y>
     <width>450</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Alien?</string>
   </property>
  </widget>
  <widget class="QLabel" name="LeftShip">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>810</y>
     <width>350</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>SHIP?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Instructions">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>760</y>
     <width>1000</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: italic 18pt &quot;Harlow Solid Italic&quot;;</string>
   </property>
   <property name="text">
    <string>Instructions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Settings">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>830</y>
     <width>1000</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
font: italic 18pt &quot;Harlow Solid Italic&quot;;</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="RightShip">
   <property name="geometry">
    <rect>
     <x>1515</x>
     <y>810</y>
     <width>350</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>SHIP?</string>
   </property>
  </widget>
  <widget class="QLabel" name="LeftBullet">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>780</y>
     <width>16</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 204, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RightBullet">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>770</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 204, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
